docs/hub-tutorials/join-mainnet.md

```diff
@@ -215,16 +189,169 @@ enable = true
 address = "0.0.0.0:9090"
 ```
 
-## Upgrades
 
-To be best prepared for eventual upgrades, it is recommended to setup [Cosmovisor](https://docs.cosmos.network/master/run-node/cosmovisor.html), a small process manager,  which can swap in new `gaiad` binaries. Read more about setting this up [here](upgrade-node.md).
+## Choosing a Sync Mode
+
+There are three main ways to sync a node on the Cosmos Hub. The way a node operator will want to sync is partially dependent on the type of node they will operate. This guide will focus on two types of common nodes; archive/full and pruned. For further information on syncing to run a validator node, see the section on [Validators](https://hub.cosmos.network/main/validators/overview.html).
 
-## Background Process
+If a node operator wishes to run a full/archive node, it is possible to start from scratch but will take a significant amount of time to catch up. Node operators not concerned with rebuilding original state from the beginning of `cosmoshub-4` can also leverage [Quicksync](#Quicksync)'s available archive history.
 
-To run the node in a background process with automatic restarts, you can use a service manager like `systemd`. To set this up run the following:
+For operators interested in bootstrapping a pruned node, either [Quicksync](#Quicksync) or [State Sync](#State-Sync) would be sufficient.
 
+Make sure to consult the [hardware](#Hardware) section for guidance on the best configuration for the type of node operating.
+
+### Sync from scratch
+
+Syncing from scratch is possible by starting the chain from the beginning of `cosmoshub-4` and either manually upgrading the chain or setting up [Cosmovisor](#Cosmovisor) to upgrade automatically. Node operators syncing from scratch will need to navigate two upgrades, Delta and Vega. For more information on performing the manual upgrades, see [Releases & Upgrades](#Releases-amp=-Upgrades).
+
+Start Gaia to begin syncing with the `skip-invariants` flag. For more information on this see [Verify Mainnet](#Verify-Mainnet).
 ```bash
-sudo tee /etc/systemd/system/gaiad.service > /dev/null <<EOF  
+gaiad start --x-crisis-skip-assert-invariants
+
+```
+
+The node will begin rebuilding state until it hits the first upgrade height at block `6910000`. If Cosmovisor is set up then there's nothing else to do besides wait, otherwise the node operator will need to perform the manual upgrade twice.
+
+### State Sync
+
+Enabling state sync can be one of the most efficient ways to bootstrap a node. On startup, the node will ask its peers for available snapshots. Once a snapshot has been accepted, it will begin requesting snapshot chunks from peers. Once the snapshot is verified, the node will begin to process blocks in realtime until it catches up with the head of the chain. For more information on state sync, see [Tendermint's documentation](https://docs.tendermint.com/master/spec/p2p/messages/state-sync.html).
```

Comment:
Statesync can apply multiple snapshots, after which the node can start using blocksync/fastsync (or consensus replay).

Having said that I might say: "statesync is an efficient way to bootstrap a new node, and it works by replaying larger chunks of application state directly rather than replaying individual blocks or consensus rounds." which is probably closer to what readers care about most?

Reply:
👍